build.gradle.kts

```diff
@@ -17,6 +19,31 @@ springBoot {
   mainClass = "com.mycompany.myapp.JhipsterSampleApplicationApp"
 }
 
+node {
+  version.set("v20.12.2")
+  npmVersion.set("10.5.2")
+}
+
+val testTaskUsingNpm = tasks.register<NpmTask>("testNpm") {
+  dependsOn("test", "integrationTest")
+  dependsOn(tasks.npmInstall)
+  npmCommand.set(listOf("run", "test"))
+//  args.set(listOf("test"))
+  ignoreExitValue.set(false)
+  environment.set(mapOf("APP_VERSION" to "0.0.1-SNAPSHOT")) //gradle doesnt use project version like maven. Where should I get that?
+  workingDir.set(projectDir)
+  execOverrides {
+    standardOutput = System.out
+  }
+  inputs.dir("node_modules")
+  inputs.file("package.json")
```

Comment:
There are a bit more input files:
- package-lock.json
- src/test/javascript
- src/main/webapp
- tsconfig*.json
- vite.config.*
- vitest.config.*

See e.g. for jhlite: https://github.com/jhipster/jhipster-lite/blob/3266169af076475fb0b00a6e0330128371414407/pom.xml#L912-L948